clicommand/agent_start.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/mitchellh/go-homedir"
 	"github.com/urfave/cli"
 	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
```

Comment:
Oh goodness. When did we start depending on `exp` packages?

At least these will be making their way into Go 1.21 largely the same.

Reply:
um, it's, um, it's [hard to say, exactly](https://github.com/buildkite/agent/commit/851073dc33c89a89f623cd0a0ce1e7b8a6394b79) when (May 5th 2022) this started or who (Benjamin J. Moskovitz) started it